### ISSUE TITLE
Add option to produce html output

### ISSUE THREAD

**Issue by mojavelinux** [asciidoctor/asciidoctor-fopub#55]
Add option to produce chunked (`chunked`) HTML and htmlhelp (`htmlhelp`). This simply involves pointing to the appropriate XSL from the run script.

* For the Unix script, https://github.com/asciidoctor/asciidoctor-fopub/blob/master/fopub#L113-L125
* For the Windows script, https://github.com/asciidoctor/asciidoctor-fopub/blob/master/fopub.bat#L65-L75

**Comment by getreu** [comment]
Indeed, this does not seem to be a big deal. I am curious to see the html output!
This feature could also be interesting for publishing scientific articles. (I remember an issue that docbook inserted the word "Section" in references and asciidoctor html5 didn't).

**Comment by mojavelinux** [comment]
>  (I remember an issue that docbook inserted the word "Section" in references and asciidoctor html5 didn't).

I'm working on this as part of Asciidoctor 1.5.6. See https://github.com/asciidoctor/asciidoctor/pull/2220
